### ISSUE TITLE
Training Disconnect and Resume Failure

### ISSUE THREAD

**Issue by hartmania** [ultralytics/hub#1229]
### Search before asking

- [x] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar bug report.

### HUB Component

Training

### Bug

Hello - model training has been disconnected and is now unable to be reconnected to finish training.  In fact, now the GUI says 'we're gearing up to launch this feature (Ultralytics Cloud Training)...

'Deploy Model' says Model Not Trained....

Trained via Ultralytics HUB Cloud Training
Model ID LdLhttSqpZsauLtJ0tgv
Workspace 'SealBlast Custom Tweaks'
Last response on 11 November 2025 at 13:13

<img width="1465" height="853" alt="Image" src="https://github.com/user-attachments/assets/84277709-0695-49d2-aff6-ca6add31669e" />

### Environment

mac, chrome browser

**Comment by pderrenger** [comment]
Hi @hartmania — thanks for reporting this and for the clear details. Sorry you hit this; the “gearing up to launch this feature” banner shouldn’t appear for an account that’s already using Ultralytics HUB Cloud Training.

What I suspect
- A transient UI gating/regression in the Cloud Training tile on the Model page, not your project. We’ve flagged your model ID LdLhttSqpZsauLtJ0tgv to the HUB Cloud team for investigation as of Nov 13, 2025. We’ll update here once we confirm the root cause.

Quick checks you can try now
- Hard refresh and re-auth: Sign out, hard refresh (Cmd/Ctrl+Shift+R), sign back in, then open your model’s page and the Train dialog again.
- Verify Pro status and balance: Cloud Training requires Pro plus a minimum US$5 account balance if you used Epochs billing; training pauses automatically when balance is insufficient and is resumable from the last checkpoint. See the Cloud Training notes on balance checks, pausing, and resuming. See the Cloud Training docs for details. ([docs.ultralytics.com](https://docs.ultralytics.com/hub/cloud-training/))
- Only one active Cloud job: If another model is training in the same account/workspace, start/Resume can be disabled. HUB supports one concurrent Cloud job per account today. ([docs.ultralytics.com](https://docs.ultralytics.com/hub/cloud-training/))

How to resume from the Model page (preferred)
- Open Models → your model → Train.
- If the session paused (e.g., balance or a transient disconnect) you should see an option to resume from the last checkpoint within the Train step. The Models page workflow and Train dialog are described here. ([docs.ultralytics.com](https://docs.ultralytics.com/hub/models/))

Fallbacks/workarounds if the UI banner persists
- Google Colab or Bring Your Own Agent: Start training via Colab or your own GPU and keep the model connected to HUB so metrics and checkpoints sync back. From the Train dialog, choose “Google Colab” or “Bring your own agent.” ([docs.ultralytics.com](https://docs.ultralytics.com/hub/models/))
- Resume locally from a checkpoint: If you have a partial checkpoint (last.pt/best.pt), you can resume with YOLO locally or in Colab:
  - CLI: yolo train resume model=path/to/last.pt
  - Python:
    from ultralytics import YOLO
    model = YOLO("path/to/last.pt")
    model.train(resume=True)
  This restores optimizer state, LR scheduler and epoch counter. ([docs.ultralytics.com](https://docs.ultralytics.com/modes/train/))

Billing and visibility
- While training or after, you can confirm any charges and download a cost report from the Billing tab on the model page. ([docs.ultralytics.com](https://docs.ultralytics.com/hub/cloud-training/))

If the problem continues, please share the following to help us pinpoint it quickly:
- Exact page where the banner appears (Models list, Project page, or the model’s Train step).
- Whether the session was Epochs or Timed, and your current account balance.
- A fresh screenshot of the browser Console and Network tabs (open DevTools, then reproduce the banner).
- Confirm whether any other model was training in your account around 13:13 on Nov 11, 2025.

References
- Cloud Training: pausing/resuming behavior, minimum balance, one active job, and billing. See the Ultralytics HUB Cloud Training docs. ([docs.ultralytics.com](https://docs.ultralytics.com/hub/cloud-training/))
- Train dialog options (Cloud/Colab/BYO) and resuming from the Model page. See Ultralytics HUB Models. ([docs.ultralytics.com](https://docs.ultralytics.com/hub/models/))
- How to resume training with YOLO from a local checkpoint. See Train mode docs. ([docs.ultralytics.com](https://docs.ultralytics.com/modes/train/))

Appreciate your patience while we look into the gating banner. If you can post the extra details here, we’ll keep the investigation moving and get you unblocked quickly. 🙏

**Comment by yogendrasinghx** [comment]
Hi @hartmania,

Thanks for sharing the Model ID.

I’ve successfully restored your training to epoch 288, and you should now be able to resume the training without any issues.

Sorry for the inconvenience!

**Comment by hartmania** [comment]
Howdy,

I have done the refreshes, sign out, new browser etc --- but I can't get
rid of the "we're gearing up to launch this feature" wall....

Can't continue training, can't start training from scratch, etc etc


[image: image.png]
[image: image.png]

**Comment by hartmania** [comment]
I take that back - switching to safari worked - THANK YOU!

**Comment by pderrenger** [comment]
Great news, and thanks for confirming Safari worked! 🙌 We also see your run restored to epoch 288 per Yogendra’s update, so you should be able to Resume from the model’s Train dialog and continue as normal. Once a new checkpoint is produced, the Deploy tab will reflect that the model has trained and enable export.

If you prefer to stay on Chrome, a quick cleanup usually clears that incorrect “we’re gearing up to launch this feature” banner (it’s a stale cache/feature-flag state):
- In Chrome, open your model page, then DevTools → Application → Service Workers → Unregister (and Update if visible).
- Still in Application → Storage → Clear site data (include Cache Storage and Local Storage), then hard-reload (Cmd/Ctrl+Shift+R) and sign in again.
- Test in an Incognito window with extensions disabled to rule out blockers.

A couple of tips while you resume:
- Only one Cloud job can run per account at a time; if another training is active, Resume/Start may be gated. See the Cloud Training notes on concurrency and pausing/resuming. See the Cloud Training documentation for details.
- If you trained with Epochs billing, make sure your account balance is ≥ US$5 so Resume doesn’t pause immediately; charges occur per epoch. The Billing tab on the model tracks costs and lets you download a cost report. See the Cloud Training billing section.
- The Models page and Train dialog flow are outlined here if you want a quick refresher. See the Ultralytics HUB Models documentation.

We’re investigating the Chrome-specific caching that surfaced here so the banner doesn’t reappear; we’ll post back once the fix is live. If you hit it again in Chrome, a short Console + Network screenshot after a fresh repro would help us nail it.

Really appreciate your patience—and kudos to the HUB team and the YOLO community for the quick restore. 🚀
